README.md

```diff
@@ -115,13 +149,26 @@ any given point in time, fully support a maximum of 2 LTS releases.
 [`NAN`]: https://github.com/nodejs/nan
 
 ## LTS Team members
-
 * Gibson Fahnestock [@gibfahn](https://github.com/gibfahn)
-* James M Snell [@jasnell](https://github.com/jasnell)
-* Jeremiah Senkpiel [@Fishrock123](https://github.com/Fishrock123)
+* James M Snell [@jasnell](https://github.com/jasnell) 
+* Jeremiah Senkpiel [@Fishrock123](https://github.com/Fishrock123) 
 * Michael Dawson [@mhdawson](https://github.com/mhdawson)
-* Myles Borins [@MylesBorins](https://github.com/MylesBorins)
-* Rod Vagg [@rvagg](https://github.com/rvagg)
+* Myles Borins [@MylesBorins](https://github.com/MylesBorins) 
 * Sam Roberts [@sam-github](https://github.com/sam-github)
 
-Github team for LTS: https://github.com/orgs/nodejs/teams/lts
+### Releasers team
```

Comment:
Nope. My apologies, will fix up in next pass through.  I messed that up willing splitting up the teams.